mobile/lib/features/trade/trade_screen.dart

```diff
@@ -62,16 +66,22 @@ class TradeScreen extends StatelessWidget {
                   }, builder: (context, price, child) {
                     return LatestPriceWidget(
                       innerKey: tradeScreenAskPrice,
-                      label: "Latest Ask: ",
+                      label: "Ask: ",
                       price: Usd.fromDouble(price ?? 0.0),
                     );
                   }),
+                  Selector<FundingRateChangeNotifier, double?>(selector: (_, provider) {
+                    return provider.nextRate;
+                  }, builder: (context, rate, child) {
+                    return FundingRateWidget(
+                        rate: rate, label: "Funding Rate: ", innerKey: tradeScreenFundingRate);
```

Comment:
🙃 A countdown to the next funding rate event would be nice.

Reply:
I'm gonna try to find a way to put this on the screen.

Reply:
I did it.